### DISCUSSION TITLE
how do I add hyper sonic to my project.

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#76] [Q&A] [answered]
I am trying to add hyper sonic but I don't know how

**Comment by Techokami** [comment, marked as answer]
To start, I would suggest studying how the super transformation currently in Worlds Next works, then adapting it with a count for Super Emeralds instead of the Chaos Emeralds. If this is too difficult, then I suggest learning from the Godot tutorials in the manual.

**Comment by DimensionWarped** [comment]
Also please stop posting questions without telling us what you've put into trying to figure out a problem so far. You need to put some effort into it before asking for help... at least enough effort to be able to detail that effort. If you haven't been diligently trying to solve a problem for at least two hours before getting stumped, it isn't really appropriate to ask for help.

I don't want to come across like I intend to punish you or anything like that, but this is the second time I've tried to explain this to you.
